test/cypress/integration/widgets/blueprints_spec.ts

```diff
@@ -389,10 +403,53 @@ describe('Blueprints widget', () => {
             cy.contains('Upload from Marketplace').click();
         });
 
-        it('should open marketplace modal', () => {
-            cy.contains('.header', 'Blueprint marketplace');
-            cy.get('.tabular > a.item').should('have.length', 3);
-            cy.get('.blueprintCatalogWidget').should('exist');
+        it('and have blueprint catalog widget', () => {
+            cy.get('.modal').within(() => {
+                cy.contains('.header', 'Blueprint marketplace');
+                cy.get('.tabular > a.item').should('have.length', 3); // 3 - default number of tabs
+                cy.get('.blueprintCatalogWidget').should('be.visible');
+            });
+        });
+    });
+
+    describe('configuration', () => {
+        it('should allow to add new marketplace tab', () => {
+            const testTabMarketplaceName = 'Blueprints from Dagobah';
+
+            cy.editWidgetConfiguration('blueprints', () => {
+                cy.contains('Add').click();
+                cy.get('input[name="name"]').eq(3).type(testTabMarketplaceName);
+                cy.get('input[name="url"]').eq(3).type('https://localhost');
+            });
+            cy.contains('Upload').click();
+            cy.contains('Upload from Marketplace').click();
+            cy.get('.modal').within(() => {
+                cy.contains(testTabMarketplaceName).should('be.visible');
+            });
```

Comment:
This is rather cosmetic, but it could be shorter:
```suggestion
            cy.contains('.modal', testTabMarketplaceName).should('be.visible');
```